cilium/cmd/config_get.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2021 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/cilium/cilium/pkg/command"
+	"github.com/cilium/cilium/test/helpers"
+	"github.com/spf13/cobra"
+)
+
+var (
+	re = regexp.MustCompile("[^0-9a-zA-Z]")
+)
+
+// configGetCmd represents the config get command
+var configGetCmd = &cobra.Command{
+	Use:    "get <config name>",
+	Short:  "Retrieve cilium configuration",
+	PreRun: requireConfigName,
+	Run: func(cmd *cobra.Command, args []string) {
+		configName := re.ReplaceAllString(args[0], "")
+		resp, err := client.ConfigGet()
+		if err != nil {
+			Fatalf("Error while retrieving configuration: %s", err)
+		}
+		if resp.Status == nil {
+			Fatalf("Empty configuration status returned")
+		}
+
+		readWriteConfigMap := make(map[string]interface{})
+		readOnlyConfigMap := resp.Status.DaemonConfigurationMap
+
+		for k, v := range resp.Status.Realized.Options {
+			readWriteConfigMap[k] = v
+		}
+		readWriteConfigMap[helpers.PolicyEnforcement] = resp.Status.Realized.PolicyEnforcement
+
+		readWriteConfigMap = mapKeysToLowerCase(readWriteConfigMap)
+		readOnlyConfigMap = mapKeysToLowerCase(readOnlyConfigMap)
+
+		configMap := mergeMaps(readOnlyConfigMap, readWriteConfigMap)
+
+		if value, ok := configMap[configName]; ok {
```

Comment:
Could you add a small comment to explain what is being done on lines 36 and 53 to 58? We've talked about it, so it's clear to me, but it may come as a bit strange to others.